mk/spksrc.python.mk

```diff
@@ -0,0 +1,49 @@
+###
+### Reuse Python libraries
+###
+
+# Default python version to use
+ifeq ($(strip $(PYTHON_VERSION)),)
+export PYTHON_VERSION = 311
+endif
```

Comment:
This will not work, we already have a variable PYTHON_VERSION that is 2.7, 3.8, 3.10 or 3.11.

found this code in spksrc.wheel.mk
```
	   elif [ "$(PYTHON_VERSION)" != "2.7" ] ; then \
```

and this error in the github build log:
```
023-07-25T12:48:07.9263087Z sed: can't read /github/workspace/spk/salt-master/../python3.11/Makefile: No such file or directory
```

Reply:
> This will not work, we already have a variable PYTHON_VERSION that is 2.7, 3.8, 3.10 or 3.11.
> 
> found this code in spksrc.wheel.mk
> ```
> 	   elif [ "$(PYTHON_VERSION)" != "2.7" ] ; then \
> ```
> 
> and this error in the github build log:
> ```
> 023-07-25T12:48:07.9263087Z sed: can't read /github/workspace/spk/salt-master/../python3.11/Makefile: No such file or directory
> ```
> 
> EDIT:
> and this is echoed to `python-cc.mk` in cross/python???/Makefile
> ```
> @echo PYTHON_VERSION=$(PKG_VERS_MAJOR_MINOR) >> $@
> ```
> 

I followed what was used for ffmpeg.  Perhaps then `PYTHON_TARGET` ?

If a specific mk file for python works out I'll reorganize ffmep in a subsequent PR to use the same workflow.